src/Database/Driver.php

```
@@ -235,6 +235,17 @@ public function supportsQuoting()
     abstract public function quoteIdentifier($identifier);
 
     /**
+     * Get the SQL for explaining the query.
+     *
+     * @param string $query The query to be explained
+     * @return string|bool
+     */
+    public function explainSQL($query)
+    {
+        return false;
```

Should throw an exception to signal that it is not implemented